### PR TITLE
[ASM] Fix boot with VS 16.7.3 / MSVC 19.27

### PR DESCRIPTION
**Pull request by tkreuzer**
See https://developercommunity.visualstudio.com/content/problem/1180800/ml-x86-assume-directive-causes-invalid-code-genera.html

Testbot results: https://reactos.org/testman/compare.php?ids=74170,74172,74181,74190

### CODE REVIEW

File: sdk/lib/crt/math/i386/pow_asm.s [hunk 1]
```diff
@@ -91,7 +91,7 @@ _pow:
 	cmp	ah, HEX(040)	// is y == 0 ?
 	je	L11
 
-	cmp ah, 5	// is y == �inf ?
+	cmp ah, 5	// is y == ±inf ?
```
Comment (HBelusca) [inline sdk/lib/crt/math/i386/pow_asm.s:94]: What changed there?

File: sdk/include/asm/asm.inc [hunk 2]
```diff
@@ -27,7 +27,7 @@ OPTION DOTNAME
 .686P
 .XMM
 .MODEL FLAT
-ASSUME CS:NOTHING, DS:NOTHING, ES:NOTHING, FS:NOTHING, GS:NOTHING
+ASSUME FS:NOTHING, GS:NOTHING
```
[the remark this answers is not in the document]
Comment (tkreuzer) [reply sdk/include/asm/asm.inc:30]: "anything"? "not anything or nothing"? "does not apply"?

File: sdk/lib/crt/math/i386/pow_asm.s [hunk 3]
```diff
@@ -107,10 +107,10 @@ _pow:
 	mov dh, ah
 	and ah, HEX(45)
 	cmp ah, HEX(040)
-	je	L20		// x is �0
+	je	L20		// x is ±0
 
 	cmp ah, 5
-	je	L15		// x is �inf
+	je	L15		// x is ±inf
```
[the remark this answers is not in the document]
Comment (tkreuzer) [reply sdk/lib/crt/math/i386/pow_asm.s:113]: Dunno, will revert

File: sdk/lib/crt/math/i386/pow_asm.s [hunk 4]
```diff
@@ -160,7 +160,7 @@ L5:	fmul st, st	// x*x : ST*x
 	fstp st		// ST*x
 	ret
 
-	/* y is �NAN */
+	/* y is ±NAN */
```
[the remark this answers is not in the document]
Comment (tkreuzer) [reply sdk/lib/crt/math/i386/pow_asm.s]: Must have been some line break magic fixed up by VS or something. Didn't show up in my diff viewer. I hard to revert the whole file and then modify the line in the diff viewer.

**Pull request merged**